index.html

```diff
@@ -0,0 +1,128 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+      <meta charset="UTF-8">
+      <meta name="viewport" content="width=device-width, initial-scale=1.0">
+      <meta http-equiv="X-UA-Compatible" content="ie=edge">
+      <link rel="stylesheet" href="style.css" type="text/css">
+      <title>Document</title>
+  </head>
+  <body>
+    <div class="wrapper">  
+      <h1>Aleksei Raketskii</h1>
+      <img src="photo.jpg" alt="me" height="400px">
```

Comment:
It's better to set both the height and the width of the image. And `px` is redundant.
[http://codeguide.academy/html-css.html#html-img-sizes](url)